.github/workflows/release.yml

```diff
@@ -0,0 +1,76 @@
+name: Run Release Please
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+
+    # Release-please creates a PR that tracks all changes
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          command: manifest
+          token: ${{secrets.RELEASE_PLEASE_ACTION_TOKEN}}
+          default-branch: main
+          signoff: "OpenFeature Bot <109696520+openfeaturebot@users.noreply.github.com>"
+    outputs:
+      release_created: ${{ steps.release.outputs.releases_created }}
+      all: ${{ toJSON(steps.release.outputs) }}
+      paths_released: ${{ steps.release.outputs.paths_released }}
+
+  # pypi-release:
```

Comment:
This is the part I'm not sure about. I believe the rest should be good to go now.

Reply:
For building, we would need to cd into each package directory, run `hatch build`  (make sure that hatch is installed first) and that should generate a `dist/` directory with the sdist .tar.gz and the .whl files.

After that the `pypa/gh-action-pypi-publish` action should be able to take over (as long as it's pointed to each package directory)

Reply:
this sounds like a good plan. @beeme1mr we can merge this first and I can adjust it accordingly. Do we also want to directly leverage trusted publishing? If yes, you would need to configure it on `pypi` similar to `python-sdk` and for each package once 🙂

Reply:
We could either merge this or work together off this draft.

I'll look into setting up `pypi` like we did for the SDK, but I have a busy schedule this week, so I'm not sure when I'll get to it.

Reply:
Hey @beeme1mr, if you don't mind, I can take over this :)